### ISSUE TITLE
details of the reconstruction loss

### ISSUE THREAD

**Issue by whq1342** [JamesQian11/TimeLDM#1]
I am curious about the details of the reconstruction loss, there are three parts, L1 loss, L2 Loss and FFT loss. I am not sure exactly how FFT loss works. Thanks!
![image](https://github.com/user-attachments/assets/d57388a8-1ab2-487f-91ce-54493b823ee3)

**Comment by whq1342** [comment]
loss_x = F.mse_loss(input_data, x_pre) + torch.mean(torch.abs(input_data - x_pre)) + torch.mean(torch.abs(torch.fft.fft(input_data, dim=2) - torch.fft.fft(x_pre, dim=2)))
